2025-03-04 14:32:03,293 INFO    MainThread:34919 [wandb_setup.py:_flush():76] Configure stats pid to 34919
2025-03-04 14:32:03,293 INFO    MainThread:34919 [wandb_setup.py:_flush():76] Loading settings from /Users/patmccarthy/.config/wandb/settings
2025-03-04 14:32:03,293 INFO    MainThread:34919 [wandb_setup.py:_flush():76] Loading settings from /Users/patmccarthy/Documents/ThalamoCortex/train_scripts/wandb/settings
2025-03-04 14:32:03,293 INFO    MainThread:34919 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-03-04 14:32:03,293 INFO    MainThread:34919 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train_scripts/wandb_test.py', 'program': '/Users/patmccarthy/Documents/ThalamoCortex/train_scripts/wandb_test.py'}
2025-03-04 14:32:03,293 INFO    MainThread:34919 [wandb_init.py:_log_setup():461] Logging user logs to /Users/patmccarthy/Documents/ThalamoCortex/train_scripts/wandb/run-20250304_143203-s0xoham6/logs/debug.log
2025-03-04 14:32:03,293 INFO    MainThread:34919 [wandb_init.py:_log_setup():462] Logging internal logs to /Users/patmccarthy/Documents/ThalamoCortex/train_scripts/wandb/run-20250304_143203-s0xoham6/logs/debug-internal.log
2025-03-04 14:32:03,294 INFO    MainThread:34919 [wandb_init.py:init():495] calling init triggers
2025-03-04 14:32:03,294 INFO    MainThread:34919 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.02, 'architecture': 'CNN', 'dataset': 'CIFAR-100', 'epochs': 10}
2025-03-04 14:32:03,294 INFO    MainThread:34919 [wandb_init.py:init():548] starting backend
2025-03-04 14:32:03,294 INFO    MainThread:34919 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-03-04 14:32:03,390 INFO    MainThread:34919 [backend.py:ensure_launched():217] starting backend process...
2025-03-04 14:32:03,399 INFO    MainThread:34919 [backend.py:ensure_launched():222] started backend process with pid: 34930
2025-03-04 14:32:03,401 INFO    MainThread:34919 [wandb_init.py:init():558] backend started and connected
2025-03-04 14:32:03,409 INFO    MainThread:34919 [wandb_init.py:init():634] updated telemetry
2025-03-04 14:32:03,423 INFO    MainThread:34919 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2025-03-04 14:32:33,442 INFO    MainThread:34919 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2025-03-04 14:32:33,457 ERROR   MainThread:34919 [wandb_init.py:init():674] backend process timed out
2025-03-04 14:32:33,462 ERROR   MainThread:34919 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2025-03-04 14:32:33,526 ERROR   MainThread:34919 [wandb_init.py:init():1070] error
Traceback (most recent call last):
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1043, in init
    run = wi.init()
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 689, in init
    backend.cleanup()
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/backend/backend.py", line 246, in cleanup
    self.interface.join()
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/interface/interface_shared.py", line 475, in join
    super().join()
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/interface/interface.py", line 666, in join
    _ = self._communicate_shutdown()
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/interface/interface_shared.py", line 472, in _communicate_shutdown
    _ = self._communicate(record)
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/Users/patmccarthy/miniconda3/envs/thalamocortex/lib/python3.10/site-packages/wandb/sdk/interface/interface_shared.py", line 231, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
